Installation issue: root@6.30.02 fails to build with --test=root option

### Steps to reproduce the issue

```console
$ spack spec -I root
Input spec
--------------------------------
 -   root

Concretized
--------------------------------
 -   root@6.30.02%gcc@12.2.0~aqua~arrow~cuda~cudnn+davix~dcache~emacs+examples~fftw~fits~fortran+gdml+gminimal~graphviz+gsl~http~ipo+math+minuit~mlp~mysql+opengl~oracle~postgres~pythia6~pythia8+python~r+roofit~root7+rpath~shadow~spectrum~sqlite~ssl+tbb+threads~tmva~tmva-cpu~tmva-gpu~tmva-pymva~tmva-sofie+unuran~vc+vdt~veccore+x+xml~xrootd build_system=cmake build_type=Release cxxstd=17 generator=make patches=22af347 arch=linux-rhel8-a64fx
[+]      ^cmake@3.26.3%gcc@12.2.0~doc+ncurses+ownlibs~qt build_system=generic build_type=Release arch=linux-rhel8-a64fx
[+]      ^davix@0.8.1%gcc@12.2.0~ipo~thirdparty build_system=cmake build_type=Release cxxstd=11 generator=make arch=linux-rhel8-a64fx
[+]          ^util-linux-uuid@2.38.1%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^fontconfig@2.14.2%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^font-util@1.4.0%gcc@12.2.0 build_system=autotools fonts=encodings,font-adobe-100dpi,font-adobe-75dpi,font-adobe-utopia-100dpi,font-adobe-utopia-75dpi,font-adobe-utopia-type1,font-alias,font-arabic-misc,font-bh-100dpi,font-bh-75dpi,font-bh-lucidatypewriter-100dpi,font-bh-lucidatypewriter-75dpi,font-bh-type1,font-bitstream-100dpi,font-bitstream-75dpi,font-bitstream-speedo,font-bitstream-type1,font-cronyx-cyrillic,font-cursor-misc,font-daewoo-misc,font-dec-misc,font-ibm-type1,font-isas-misc,font-jis-misc,font-micro-misc,font-misc-cyrillic,font-misc-ethiopic,font-misc-meltho,font-misc-misc,font-mutt-misc,font-schumacher-misc,font-screen-cyrillic,font-sun-misc,font-winitzki-cyrillic,font-xfree86-type1 arch=linux-rhel8-a64fx
[e]              ^autoconf@2.69%gcc@12.2.0 build_system=autotools patches=7793209 arch=linux-rhel8-a64fx
[e]              ^automake@1.16.1%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^bdftopcf@1.1%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]                  ^fontsproto@2.1.3%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]                  ^libxfont@1.5.4%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^mkfontdir@1.0.7%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^mkfontscale@1.2.2%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]                  ^libfontenc@1.1.7%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^gnuconfig@2022-09-17%gcc@12.2.0 build_system=generic arch=linux-rhel8-a64fx
[+]          ^gperf@3.1%gcc@12.2.0 build_system=autotools patches=3dd36db arch=linux-rhel8-a64fx
[+]      ^freetype@2.11.1%gcc@12.2.0+pic+shared build_system=autotools arch=linux-rhel8-a64fx
[+]          ^bzip2@1.0.8%gcc@12.2.0~debug~pic+shared build_system=generic arch=linux-rhel8-a64fx
[e]              ^diffutils@3.6%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^ftgl@2.4.0%gcc@12.2.0~ipo+shared build_system=cmake build_type=Release generator=make patches=001908e arch=linux-rhel8-a64fx
[+]      ^gcc-runtime@12.2.0%gcc@12.2.0 build_system=generic arch=linux-rhel8-a64fx
[+]      ^gl2ps@1.4.2%gcc@12.2.0~doc~ipo+png+zlib build_system=cmake build_type=Release generator=make arch=linux-rhel8-a64fx
[+]      ^glew@2.2.0%gcc@12.2.0~ipo build_system=cmake build_type=Release generator=make gl=glx patches=7992e52 arch=linux-rhel8-a64fx
[+]      ^glx@1.4%gcc@12.2.0 build_system=bundle arch=linux-rhel8-a64fx
[+]          ^mesa@23.2.1%gcc@12.2.0+glx+llvm+opengl~opengles+osmesa~strip build_system=meson buildtype=release default_library=shared arch=linux-rhel8-a64fx
[+]              ^bison@3.8.2%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[e]                  ^m4@1.4.18%gcc@12.2.0+sigsegv build_system=autotools patches=3877ab5,fc9b616 arch=linux-rhel8-a64fx
[+]              ^flex@2.6.3%gcc@12.2.0+lex~nls build_system=autotools arch=linux-rhel8-a64fx
[+]                  ^findutils@4.9.0%gcc@12.2.0 build_system=autotools patches=440b954 arch=linux-rhel8-a64fx
[+]              ^glproto@1.4.17%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^libunwind@1.6.2%gcc@12.2.0~block_signals~conservative_checks~cxx_exceptions~debug~debug_frame+docs~pic+tests+weak_backtrace~xz~zlib build_system=autotools components=none libs=shared,static arch=linux-rhel8-a64fx
[+]              ^libxt@1.3.0%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^llvm@17.0.4%gcc@12.2.0+clang~cuda~flang+gold~ipo+libomptarget~libomptarget_debug~link_llvm_dylib+lld+lldb+llvm_dylib+lua~mlir+polly~python~split_dwarf~z3~zstd build_system=cmake build_type=Release compiler-rt=runtime generator=ninja libcxx=runtime libunwind=runtime openmp=runtime shlib_symbol_version=none targets=all version_suffix=none arch=linux-rhel8-a64fx
[+]                  ^binutils@2.41%gcc@12.2.0~gas+gold~gprofng+headers~interwork+ld~libiberty~lto~nls~pgo+plugins build_system=autotools compress_debug_sections=zlib libs=shared,static arch=linux-rhel8-a64fx
[+]                  ^libedit@3.1-20210216%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]                  ^lua@5.3.6%gcc@12.2.0~pcfile+shared build_system=makefile fetcher=curl arch=linux-rhel8-a64fx
[e]                      ^curl@7.61.1%gcc@12.2.0+gssapi+ldap~libidn2~librtmp~libssh~libssh2+nghttp2 build_system=autotools libs=shared,static tls=openssl arch=linux-rhel8-a64fx
[+]                      ^unzip@6.0%gcc@12.2.0 build_system=makefile arch=linux-rhel8-a64fx
[+]                  ^perl-data-dumper@2.173%gcc@12.2.0 build_system=perl arch=linux-rhel8-a64fx
[+]                  ^swig@4.1.1%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]                      ^pcre2@10.42%gcc@12.2.0~jit+multibyte build_system=autotools arch=linux-rhel8-a64fx
[+]              ^meson@1.2.2%gcc@12.2.0 build_system=python_pip patches=0f0b1bd,ae59765 arch=linux-rhel8-a64fx
[+]                  ^py-pip@23.1.2%gcc@12.2.0 build_system=generic arch=linux-rhel8-a64fx
[+]                  ^py-setuptools@68.0.0%gcc@12.2.0 build_system=generic arch=linux-rhel8-a64fx
[+]                  ^py-wheel@0.41.2%gcc@12.2.0 build_system=generic arch=linux-rhel8-a64fx
[+]              ^ninja@1.11.1%gcc@12.2.0+re2c build_system=generic arch=linux-rhel8-a64fx
[+]                  ^re2c@2.2%gcc@12.2.0 build_system=generic arch=linux-rhel8-a64fx
[+]              ^py-mako@1.2.4%gcc@12.2.0 build_system=python_pip arch=linux-rhel8-a64fx
[+]                  ^py-markupsafe@2.1.3%gcc@12.2.0 build_system=python_pip arch=linux-rhel8-a64fx
[+]              ^xrandr@1.5.2%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]                  ^libxrandr@1.5.4%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]                  ^randrproto@1.5.0%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^gmake@4.4.1%gcc@12.2.0~guile build_system=generic arch=linux-rhel8-a64fx
[+]      ^gsl@2.7.1%gcc@12.2.0~external-cblas+pic+shared build_system=autotools arch=linux-rhel8-a64fx
[+]      ^intel-tbb@2021.9.0%gcc@12.2.0~ipo+shared+tm build_system=cmake build_type=Release cxxstd=default generator=make patches=91755c6 arch=linux-rhel8-a64fx
[+]          ^hwloc@2.9.1%gcc@12.2.0~cairo~cuda~gl~libudev+libxml2~netloc~nvml~oneapi-level-zero~opencl+pci~rocm build_system=autotools libs=shared,static arch=linux-rhel8-a64fx
[+]              ^libpciaccess@0.17%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[e]                  ^libtool@2.4.6%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^libice@1.1.1%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^util-macros@1.19.3%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^xtrans@1.5.0%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^libjpeg-turbo@3.0.0%gcc@12.2.0~ipo~jpeg8~partial_decoder+shared+static build_system=cmake build_type=Release generator=make arch=linux-rhel8-a64fx
[+]          ^nasm@2.15.05%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^libpng@1.6.39%gcc@12.2.0~ipo~pic build_system=cmake build_type=Release generator=make libs=shared,static arch=linux-rhel8-a64fx
[e]          ^gmake@4.2.1%gcc@12.2.0~guile build_system=autotools patches=ca60bd9,fe5b60d arch=linux-rhel8-a64fx
[+]      ^libsm@1.2.4%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^libx11@1.8.7%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^inputproto@2.3.2%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^kbproto@1.0.7%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^libxcb@1.16%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^libpthread-stubs@0.4%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^libxau@1.0.11%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^libxdmcp@1.1.4%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^xcb-proto@1.16.0%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^perl@5.36.0%gcc@12.2.0+cpanm+open+shared+threads build_system=generic arch=linux-rhel8-a64fx
[+]              ^berkeley-db@18.1.40%gcc@12.2.0+cxx~docs+stl build_system=autotools patches=26090f4,b231fcc arch=linux-rhel8-a64fx
[+]      ^libxext@1.3.5%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^libxft@2.3.8%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^libxrender@0.9.11%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]              ^renderproto@0.11.1%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^libxml2@2.10.3%gcc@12.2.0~python build_system=autotools arch=linux-rhel8-a64fx
[+]          ^libiconv@1.17%gcc@12.2.0 build_system=autotools libs=shared,static arch=linux-rhel8-a64fx
[+]      ^libxpm@3.5.17%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^gettext@0.21.1%gcc@12.2.0+bzip2+curses+git~libunistring+libxml2+tar+xz build_system=autotools arch=linux-rhel8-a64fx
[e]              ^tar@1.30%gcc@12.2.0 build_system=autotools zip=pigz arch=linux-rhel8-a64fx
[+]      ^lz4@1.9.4%gcc@12.2.0 build_system=makefile libs=shared,static arch=linux-rhel8-a64fx
[+]      ^mesa-glu@9.0.2%gcc@12.2.0 build_system=autotools gl=glx patches=3d03e55 arch=linux-rhel8-a64fx
[+]      ^ncurses@6.4%gcc@12.2.0~symlinks+termlib abi=none build_system=autotools arch=linux-rhel8-a64fx
[+]      ^nlohmann-json@3.11.2%gcc@12.2.0~ipo+multiple_headers build_system=cmake build_type=Release generator=make arch=linux-rhel8-a64fx
[+]      ^openssl@1.1.1t%gcc@12.2.0~docs~shared build_system=generic certs=mozilla arch=linux-rhel8-a64fx
[+]          ^ca-certificates-mozilla@2023-01-10%gcc@12.2.0 build_system=generic arch=linux-rhel8-a64fx
[+]      ^pcre@8.45%gcc@12.2.0~jit+multibyte+pic+shared+static+utf build_system=autotools arch=linux-rhel8-a64fx
[e]      ^pkgconf@1.4.2%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^python@3.10.10%gcc@12.2.0+bz2+crypt+ctypes+dbm~debug+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tkinter+uuid+zlib build_system=generic patches=0d98e93,7d40923,f2fd060 arch=linux-rhel8-a64fx
[+]          ^expat@2.5.0%gcc@12.2.0+libbsd build_system=autotools arch=linux-rhel8-a64fx
[+]              ^libbsd@0.11.7%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]                  ^libmd@1.0.4%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^gdbm@1.23%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^libffi@3.4.4%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]          ^libxcrypt@4.4.33%gcc@12.2.0~obsolete_api build_system=autotools arch=linux-rhel8-a64fx
[+]          ^readline@8.2%gcc@12.2.0 build_system=autotools patches=bbf97f1 arch=linux-rhel8-a64fx
[+]          ^sqlite@3.42.0%gcc@12.2.0+column_metadata+dynamic_extensions+fts~functions+rtree build_system=autotools arch=linux-rhel8-a64fx
[+]      ^unuran@1.8.1%gcc@12.2.0~gsl+rngstreams+shared build_system=autotools arch=linux-rhel8-a64fx
[+]          ^rngstreams@1.0.1%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^vdt@0.4.4%gcc@12.2.0~ipo~preload build_system=cmake build_type=Release generator=make arch=linux-rhel8-a64fx
[+]      ^xextproto@7.3.0%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^xproto@7.0.31%gcc@12.2.0 build_system=autotools arch=linux-rhel8-a64fx
[+]      ^xxhash@0.8.1%gcc@12.2.0 build_system=makefile arch=linux-rhel8-a64fx
[+]      ^xz@5.4.1%gcc@12.2.0~pic build_system=autotools libs=shared,static arch=linux-rhel8-a64fx
[+]      ^zlib@1.2.13%gcc@12.2.0+optimize+pic+shared build_system=makefile arch=linux-rhel8-a64fx
[+]      ^zstd@1.5.5%gcc@12.2.0~programs build_system=makefile libs=shared,static arch=linux-rhel8-a64fx

```


### Error message

<details><summary>Error message</summary>

<pre>
==> Installing root-6.30.02-ifueiwdgknhz2zyg3olctrljdchct3mv [80/80]
==> No binary for root-6.30.02-ifueiwdgknhz2zyg3olctrljdchct3mv found: installing from source
==> Fetching https://root.cern/download/root_v6.30.02.source.tar.gz
==> Applied patch /fefs/spack2/spack/var/spack/repos/builtin/packages/root/format-stringbuf-size.patch
==> Ran patch() for root
==> root: Executing phase: 'cmake'
==> Error: ProcessError: Command exited with status 1:
    '/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/cmake-3.26.3-vmwdkb3r4mkkfy7oqf6l4ywoorim3pqz/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/root-6.30.02-ifueiwdgknhz2zyg3olctrljdchct3mv' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/root-6.30.02-ifueiwdgknhz2zyg3olctrljdchct3mv/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/root-6.30.02-ifueiwdgknhz2zyg3olctrljdchct3mv/lib64;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/gcc-runtime-12.2.0-qqxocxya54wb3swxalgbbrorrn2lbw4r/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxml2-2.10.3-cxbpyhowuc4j6we65na4wazk5h57hjaw/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libiconv-1.17-l2yfikxxwy4kiei2dgdknjyajfrlq6hd/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xz-5.4.1-75e7je3oawrdr7cts3g3pkac5u3ftrmw/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/zlib-1.2.13-kotx3wica5v45zn7oe4m63z3pnnaexln/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/openssl-1.1.1t-4hmanhwxzsltefcszhcs5hoh6p6yhaxz/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/util-linux-uuid-2.38.1-anmpufh4gkyyfkcnynvitlpyv7gf3u6u/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/freetype-2.11.1-chjsbv2ymof57m3uqvsjau7wrq3khxfw/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/bzip2-1.0.8-lhcxas5vza3n6mltbusayf5pzkyd2gmg/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libpng-1.6.39-iepc5at2dkhrrbipz2ogjw2cdsw56vrv/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/ftgl-2.4.0-nyiy4fsdc6adc4jld35navnkpl6dsfzr/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/mesa-23.2.1-o5u62int5bps4pd7ltee5kz32z4mpps2/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/expat-2.5.0-62whkr5o55x4b57nbyikb3on7gqjxsjx/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libbsd-0.11.7-qcouol6zw6oet4b26jgh56x7i65qrr67/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libmd-1.0.4-4f7qkajfixhzajcu2fbjbept5xpbjdnn/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/glproto-1.4.17-yscy53ar23merwwassbdwqgck675b7rj/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libunwind-1.6.2-4st2xnmcar6vee2bybxsahx5mmyaldyj/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libx11-1.8.7-hp3pn3bgkusxeelmpbxuwwvdyh6lgypd/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/inputproto-2.3.2-pb6uyvsdseqfr3n57hotct36gaypd2oj/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/kbproto-1.0.7-jmngvytrijco43w5obmsenpawteufygk/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxcb-1.16-uptp5nnp56ncykycpsq5uhcuvhnui5ay/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libpthread-stubs-0.4-z3ej6jlkhgjevv7vnpvk2t5t6su3emna/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxau-1.0.11-365mqmyexahd77qmsnpo2vp24xjw7chd/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xproto-7.0.31-xybavzw24tx7tlsmj5pn5i4hciu25s6z/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxdmcp-1.1.4-qfeskktjmmg52mkqna5wzxq2a4ey74zl/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xcb-proto-1.16.0-zbclbtkgvfi6oa2nnqo4liyj3rwffase/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xextproto-7.3.0-2es5zzban7ro7dkzisuvt6wec3iksmap/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxext-1.3.5-uyhnerdqq2j4huvkfzxrsz5nwyubjigc/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxt-1.3.0-bsadpadkph57k3un2in35nmxvnu7a5hh/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libice-1.1.1-nxr7ehjhrxh7yln5rg4b66abe7sc6nqs/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libsm-1.2.4-v4yovcsoz64avoiivylviftvz4azlues/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/llvm-17.0.4-mqzvjbwtpyvopis7klc7tgzaef2al3pk/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/binutils-2.41-nr5zefzjj6nyadlrh4denbuyhhfjzwrr/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/zstd-1.5.5-sjpndirj6xoqz34lqwiyporheviux43m/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/hwloc-2.9.1-f4afpxxpxmqlthpcirhh5mptd2m355j7/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libpciaccess-0.17-yvkpfxdjluomyaa5j33dldxd2f5pyxmy/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/ncurses-6.4-plr4ufeutvh57vpvdjhu2ecw3ldtxlr4/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libedit-3.1-20210216-mldy7glkc7yksv6nisuw6pj4yryly657/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libffi-3.4.4-jp5lxx6hms5s7ghheuckwdh6jds5kkc3/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/lua-5.3.6-r33jf6lecdh7opm5rjnok3isigijiuir/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/readline-8.2-p6l4qudpynq7eoiyzvzn2h5h5inp2isr/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/pcre2-10.42-tww2soeyz6bmxdjklpmhvf7l5lrugqsx/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxrandr-1.5.4-yujdd4kwgwjmvzet3hsplef7maa2peqg/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxrender-0.9.11-jeqhda7dfduqk2t2tpvmksdk2kowx2ey/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/renderproto-0.11.1-f7pynkx33ptgyhngtf3ybqhck2lmjf3m/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/randrproto-1.5.0-fijd6mduxpsxdazwmcd6begegt7hfzut/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/mesa-glu-9.0.2-vtz3pzoz747okdfosat7acje6jd43gz7/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/gl2ps-1.4.2-uzbycrejpp7ockwm35rmlec474g7opx6/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/gsl-2.7.1-zhkf5ntkgqkdw6j5ipyqf7ry7bmxpbji/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxft-2.3.8-nqzxyxy2jhfnhtnsrdqpzuypqh4vqlee/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/fontconfig-2.14.2-fh3upe3jrlm2wlmf57bldgjcdnvb7lbg/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/font-util-1.4.0-wvsus72r73gxg4ozlzgkflmbvgp5wmh6/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxpm-3.5.17-rus5yllilj5syin7yticenyggxfmn4iq/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/gettext-0.21.1-sisiz7izg4hfeoqmwzd6ff2t7fjb44kv/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/lz4-1.9.4-g3vatbvvmhfiwjf4dzf2rglwuvhhht6y/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/pcre-8.45-pabgx3gr6jtvviwsqxlrvgcgny4navdh/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/unuran-1.8.1-jmddpsvpiqy5bhvgod4fnlfhfizvwys4/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/rngstreams-1.0.1-dlovkj6k4cuqh7fh55nxdukn2aao5x4g/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/vdt-0.4.4-wsth5zvv2nrycjse73f5d2vjhd4az6j3/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xxhash-0.8.1-rjdjivho5qxn6ks76wkw2p7kcecw2c57/lib;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/davix-0.8.1-57rrh2cb474hxhwwl2miooe5czzyb4u2/lib64;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libpng-1.6.39-iepc5at2dkhrrbipz2ogjw2cdsw56vrv/lib64;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libffi-3.4.4-jp5lxx6hms5s7ghheuckwdh6jds5kkc3/lib64;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/glew-2.2.0-rdskl4kleq2l6wg3v35delwosol6vjsi/lib64;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/intel-tbb-2021.9.0-gdaogjyvbu5aa4ildhcle2opoj6fwc33/lib64;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libjpeg-turbo-3.0.0-4qtvqsejf7carhrayh62cycvt73zrnvq/lib64' '-DCMAKE_PREFIX_PATH:STRING=/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xxhash-0.8.1-rjdjivho5qxn6ks76wkw2p7kcecw2c57;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/vdt-0.4.4-wsth5zvv2nrycjse73f5d2vjhd4az6j3;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/unuran-1.8.1-jmddpsvpiqy5bhvgod4fnlfhfizvwys4;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/rngstreams-1.0.1-dlovkj6k4cuqh7fh55nxdukn2aao5x4g;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/pcre-8.45-pabgx3gr6jtvviwsqxlrvgcgny4navdh;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/nlohmann-json-3.11.2-r4ltcqq2af6n6kuxopsfrrqfgrkhas37;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/lz4-1.9.4-g3vatbvvmhfiwjf4dzf2rglwuvhhht6y;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxpm-3.5.17-rus5yllilj5syin7yticenyggxfmn4iq;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxft-2.3.8-nqzxyxy2jhfnhtnsrdqpzuypqh4vqlee;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libjpeg-turbo-3.0.0-4qtvqsejf7carhrayh62cycvt73zrnvq;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/intel-tbb-2021.9.0-gdaogjyvbu5aa4ildhcle2opoj6fwc33;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/gsl-2.7.1-zhkf5ntkgqkdw6j5ipyqf7ry7bmxpbji;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/glew-2.2.0-rdskl4kleq2l6wg3v35delwosol6vjsi;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/gl2ps-1.4.2-uzbycrejpp7ockwm35rmlec474g7opx6;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/ftgl-2.4.0-nyiy4fsdc6adc4jld35navnkpl6dsfzr;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/mesa-glu-9.0.2-vtz3pzoz747okdfosat7acje6jd43gz7;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/glx-1.4-bvsyoioazvyw6nw6s74dlcobu3et6wvx;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/mesa-23.2.1-o5u62int5bps4pd7ltee5kz32z4mpps2;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xrandr-1.5.2-z3fywfmqoss7q5v2jzncxhuzvgxsnlle;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxrandr-1.5.4-yujdd4kwgwjmvzet3hsplef7maa2peqg;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/randrproto-1.5.0-fijd6mduxpsxdazwmcd6begegt7hfzut;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxrender-0.9.11-jeqhda7dfduqk2t2tpvmksdk2kowx2ey;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/renderproto-0.11.1-f7pynkx33ptgyhngtf3ybqhck2lmjf3m;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/llvm-17.0.4-mqzvjbwtpyvopis7klc7tgzaef2al3pk;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/swig-4.1.1-5eu26z3p66zknedcvpkzrlty3eh4lo4f;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/pcre2-10.42-tww2soeyz6bmxdjklpmhvf7l5lrugqsx;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/lua-5.3.6-r33jf6lecdh7opm5rjnok3isigijiuir;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libedit-3.1-20210216-mldy7glkc7yksv6nisuw6pj4yryly657;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/hwloc-2.9.1-f4afpxxpxmqlthpcirhh5mptd2m355j7;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libpciaccess-0.17-yvkpfxdjluomyaa5j33dldxd2f5pyxmy;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/binutils-2.41-nr5zefzjj6nyadlrh4denbuyhhfjzwrr;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/zstd-1.5.5-sjpndirj6xoqz34lqwiyporheviux43m;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxt-1.3.0-bsadpadkph57k3un2in35nmxvnu7a5hh;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libsm-1.2.4-v4yovcsoz64avoiivylviftvz4azlues;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libice-1.1.1-nxr7ehjhrxh7yln5rg4b66abe7sc6nqs;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxext-1.3.5-uyhnerdqq2j4huvkfzxrsz5nwyubjigc;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libx11-1.8.7-hp3pn3bgkusxeelmpbxuwwvdyh6lgypd;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xextproto-7.3.0-2es5zzban7ro7dkzisuvt6wec3iksmap;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxcb-1.16-uptp5nnp56ncykycpsq5uhcuvhnui5ay;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xcb-proto-1.16.0-zbclbtkgvfi6oa2nnqo4liyj3rwffase;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxdmcp-1.1.4-qfeskktjmmg52mkqna5wzxq2a4ey74zl;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxau-1.0.11-365mqmyexahd77qmsnpo2vp24xjw7chd;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libpthread-stubs-0.4-z3ej6jlkhgjevv7vnpvk2t5t6su3emna;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/kbproto-1.0.7-jmngvytrijco43w5obmsenpawteufygk;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/inputproto-2.3.2-pb6uyvsdseqfr3n57hotct36gaypd2oj;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libunwind-1.6.2-4st2xnmcar6vee2bybxsahx5mmyaldyj;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/glproto-1.4.17-yscy53ar23merwwassbdwqgck675b7rj;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/fontconfig-2.14.2-fh3upe3jrlm2wlmf57bldgjcdnvb7lbg;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/python-3.10.10-rphmd4ze2pcb4yuarionuk6ih5hszzeq;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libffi-3.4.4-jp5lxx6hms5s7ghheuckwdh6jds5kkc3;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/gettext-0.21.1-sisiz7izg4hfeoqmwzd6ff2t7fjb44kv;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/expat-2.5.0-62whkr5o55x4b57nbyikb3on7gqjxsjx;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libbsd-0.11.7-qcouol6zw6oet4b26jgh56x7i65qrr67;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libmd-1.0.4-4f7qkajfixhzajcu2fbjbept5xpbjdnn;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/font-util-1.4.0-wvsus72r73gxg4ozlzgkflmbvgp5wmh6;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xtrans-1.5.0-fhtpqhvoqnynyhcbdbge5l55m6w4mvzh;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xproto-7.0.31-xybavzw24tx7tlsmj5pn5i4hciu25s6z;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/freetype-2.11.1-chjsbv2ymof57m3uqvsjau7wrq3khxfw;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libpng-1.6.39-iepc5at2dkhrrbipz2ogjw2cdsw56vrv;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/davix-0.8.1-57rrh2cb474hxhwwl2miooe5czzyb4u2;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/util-linux-uuid-2.38.1-anmpufh4gkyyfkcnynvitlpyv7gf3u6u;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libxml2-2.10.3-cxbpyhowuc4j6we65na4wazk5h57hjaw;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/xz-5.4.1-75e7je3oawrdr7cts3g3pkac5u3ftrmw;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/libiconv-1.17-l2yfikxxwy4kiei2dgdknjyajfrlq6hd;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/gmake-4.4.1-haz65wva5fb267o6iesazxavblbey4ba;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/gcc-runtime-12.2.0-qqxocxya54wb3swxalgbbrorrn2lbw4r;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/cmake-3.26.3-vmwdkb3r4mkkfy7oqf6l4ywoorim3pqz;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/openssl-1.1.1t-4hmanhwxzsltefcszhcs5hoh6p6yhaxz;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/zlib-1.2.13-kotx3wica5v45zn7oe4m63z3pnnaexln;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/readline-8.2-p6l4qudpynq7eoiyzvzn2h5h5inp2isr;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/bzip2-1.0.8-lhcxas5vza3n6mltbusayf5pzkyd2gmg;/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/ncurses-6.4-plr4ufeutvh57vpvdjhu2ecw3ldtxlr4' '-Dexceptions:BOOL=ON' '-Dexplicitlink:BOOL=ON' '-Dfail-on-missing:BOOL=ON' '-Dfortran:BOOL=OFF' '-Dgminimal:BOOL=ON' '-Dgnuinstall:BOOL=ON' '-Dlibcxx:BOOL=OFF' '-Dpch:BOOL=ON' '-Droottest:BOOL=OFF' '-Drpath:BOOL=ON' '-Druntime_cxxmodules:BOOL=OFF' '-Dshared:BOOL=ON' '-Dsoversion:BOOL=ON' '-Dtesting:BOOL=ON' '-Dthread:BOOL=ON' '-DCLING_CXX_PATH:STRING=/fefs/llvm/spack/spack/opt/spack/linux-rhel8-a64fx/gcc-8.5.0/gcc-12.2.0-4cmifaftyfsyftrpwijuuzyion6pcf7x/bin/g++' '-Dbuiltin_afterimage:BOOL=ON' '-Dbuiltin_cfitsio:BOOL=OFF' '-Dbuiltin_davix:BOOL=OFF' '-Dbuiltin_fftw3:BOOL=OFF' '-Dbuiltin_freetype:BOOL=OFF' '-Dbuiltin_ftgl:BOOL=OFF' '-Dbuiltin_gl2ps:BOOL=OFF' '-Dbuiltin_glew:BOOL=OFF' '-Dbuiltin_gsl:BOOL=OFF' '-Dbuiltin_llvm:BOOL=ON' '-Dbuiltin_lz4:BOOL=OFF' '-Dbuiltin_lzma:BOOL=OFF' '-Dbuiltin_nlohmannjson:BOOL=OFF' '-Dbuiltin_openssl:BOOL=OFF' '-Dbuiltin_pcre:BOOL=OFF' '-Dbuiltin_tbb:BOOL=OFF' '-Dbuiltin_unuran:BOOL=OFF' '-Dbuiltin_vc:BOOL=OFF' '-Dbuiltin_vdt:BOOL=OFF' '-Dbuiltin_veccore:BOOL=OFF' '-Dbuiltin_xrootd:BOOL=OFF' '-Dbuiltin_xxhash:BOOL=OFF' '-Dbuiltin_zlib:BOOL=OFF' '-Dafdsmrgd:BOOL=OFF' '-Dafs:BOOL=OFF' '-Dalien:BOOL=OFF' '-Darrow:BOOL=OFF' '-Dasimage:BOOL=ON' '-Dastiff:BOOL=ON' '-Dbonjour:BOOL=OFF' '-Dcastor:BOOL=OFF' '-Dccache:BOOL=OFF' '-Dchirp:BOOL=OFF' '-Dcling:BOOL=ON' '-Dcocoa:BOOL=OFF' '-Ddataframe:BOOL=ON' '-Ddavix:BOOL=ON' '-Ddcache:BOOL=OFF' '-Dfftw3:BOOL=OFF' '-Dfitsio:BOOL=OFF' '-Dftgl:BOOL=ON' '-Dgdml:BOOL=ON' '-Dgenvector:BOOL=ON' '-Dgeocad:BOOL=OFF' '-Dgfal:BOOL=OFF' '-Dgl2ps:BOOL=ON' '-Dglite:BOOL=OFF' '-Dglobus:BOOL=OFF' '-Dgviz:BOOL=OFF' '-Dhdfs:BOOL=OFF' '-Dhttp:BOOL=OFF' '-Dimt:BOOL=ON' '' '-Dkrb5:BOOL=OFF' '-Dldap:BOOL=OFF' '-Dmathmore:BOOL=ON' '' '-Dminimal:BOOL=OFF' '-Dminuit:BOOL=ON' '-Dminuit2:BOOL=ON' '-Dmlp:BOOL=OFF' '-Dmonalisa:BOOL=OFF' '-Dmysql:BOOL=OFF' '-Dodbc:BOOL=OFF' '-Dopengl:BOOL=ON' '-Doracle:BOOL=OFF' '-Dpgsql:BOOL=OFF' '-Dpythia6:BOOL=OFF' '-Dpythia8:BOOL=OFF' '' '' '-Dr:BOOL=OFF' '-Drfio:BOOL=OFF' '-Droofit:BOOL=ON' '-Droot7:BOOL=OFF' '-Druby:BOOL=OFF' '-Dsapdb:BOOL=OFF' '-Dshadowpw:BOOL=OFF' '-Dspectrum:BOOL=OFF' '-Dsqlite:BOOL=OFF' '-Dsrp:BOOL=OFF' '-Dssl:BOOL=OFF' '' '-Dtbb:BOOL=ON' '-Dtcmalloc:BOOL=OFF' '-Dtmva:BOOL=OFF' '-Dunuran:BOOL=ON' '-Dvc:BOOL=OFF' '-Dvdt:BOOL=ON' '-Dveccore:BOOL=OFF' '' '-Dx11:BOOL=ON' '-Dxft:BOOL=ON' '-Dxml:BOOL=ON' '-Dxrootd:BOOL=OFF' '-Dcuda:BOOL=OFF' '' '-Dtmva-cpu:BOOL=OFF' '-Dtmva-gpu:BOOL=OFF' '-Dtmva-pymva:BOOL=OFF' '-Dcudnn:BOOL=OFF' '-Dpyroot:BOOL=ON' '-Dtmva-sofie:BOOL=OFF' '-DCMAKE_CXX_STANDARD:STRING=17' '-DFTGL_ROOT_DIR:STRING=/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/ftgl-2.4.0-nyiy4fsdc6adc4jld35navnkpl6dsfzr' '-DFTGL_INCLUDE_DIR:STRING=/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/ftgl-2.4.0-nyiy4fsdc6adc4jld35navnkpl6dsfzr/include' '-DPYTHON_EXECUTABLE:STRING=/fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/python-3.10.10-rphmd4ze2pcb4yuarionuk6ih5hszzeq/bin/python3.10' '/fefs/spack2/tmp/spack-stage/spack-stage-root-6.30.02-ifueiwdgknhz2zyg3olctrljdchct3mv/spack-src'

1 error found in build log:
     148    -- Looking for uuid_generate_random in uuid - found
     149    -- Found libuuid: /fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/util-linux-uuid-2.38.1-anmpufh4gkyyfkcnynvitlpyv7gf3u6u/include
     150    -- Looking for TBB
     151    -- Found TBB: /fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/intel-tbb-2021.9.0-gdaogjyvbu5aa4ildhcle2opoj6fwc33 (Required is at least ve
            rsion "2018")
     152    -- Looking for VDT
     153    -- Found Vdt: /fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/vdt-0.4.4-wsth5zvv2nrycjse73f5d2vjhd4az6j3/include (found suitable version "
            0.4", minimum required is "0.4")
  >> 154    CMake Error at /fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/cmake-3.26.3-vmwdkb3r4mkkfy7oqf6l4ywoorim3pqz/share/cmake-3.26/Modules/Find
            PackageHandleStandardArgs.cmake:230 (message):
     155      Could NOT find GTest (missing: GTEST_LIBRARIES GTEST_MAIN_LIBRARIES
     156      GTEST_INCLUDE_DIRS GMOCK_LIBRARIES GMOCK_MAIN_LIBRARIES GMOCK_INCLUDE_DIRS)
     157    Call Stack (most recent call first):
     158      /fefs/spack2/spack/opt/spack/linux-rhel8-a64fx/gcc-12.2.0/cmake-3.26.3-vmwdkb3r4mkkfy7oqf6l4ywoorim3pqz/share/cmake-3.26/Modules/FindPackageHandle
            StandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
     159      cmake/modules/FindGTest.cmake:75 (find_package_handle_standard_args)
     160      cmake/modules/SearchInstalledSoftware.cmake:18 (_find_package)

See build log for details:
  /fefs/spack2/tmp/spack-stage/spack-stage-root-6.30.02-ifueiwdgknhz2zyg3olctrljdchct3mv/spack-build-out.txt

</pre></details>


### Information on your system

* **Spack:** 0.22.0.dev0 (5f58a4c0792071ca4e681e43724fe7c66c2cefce)
* **Python:** 3.6.8
* **Platform:** linux-rhel8-a64fx
* **Concretizer:** clingo

### Additional information

Hi maintainers(@HadrienG2  @drbenmorgan  @gartung  @greenc-FNAL  @marcmengel  @vitodb  @vvolkl)
When I built root@6.30.02 with the --test=root option, cmake encountered an error.
The error message probably indicates that googletest is not listed as a dependency in package.py.
So I modified the root app package.py as follows, and cmake succeeded.

```diff
diff --git a/var/spack/repos/builtin/packages/root/package.py b/var/spack/repos/builtin/packages/root/package.py
index cec503f4cf..b8136cac8f 100644
--- a/var/spack/repos/builtin/packages/root/package.py
+++ b/var/spack/repos/builtin/packages/root/package.py
@@ -345,7 +345,7 @@ class Root(CMakePackage):
     depends_on("libxml2", when="+xml")
     depends_on("xrootd", when="+xrootd")
     depends_on("xrootd@:4", when="@:6.22.03 +xrootd")
-
+    depends_on("googletest", type="test")
     # ###################### Conflicts ######################
 
     # I was unable to build root with any Intel compiler
```

* build log
[spack-build-env.txt](https://github.com/spack/spack/files/14309616/spack-build-env.txt)
[spack-build-out.txt](https://github.com/spack/spack/files/14309618/spack-build-out.txt)

* Fixes for package.py
[root_package.patch](https://github.com/spack/spack/files/14309806/root_package.patch)


### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/root/package.py
+++ b/var/spack/repos/builtin/packages/root/package.py
@@ -352,6 +352,8 @@ class Root(CMakePackage):
     depends_on("xrootd", when="+xrootd")
     depends_on("xrootd@:4", when="@:6.22.03 +xrootd")
 
+    depends_on("googletest", when="@6.28.00:", type="test")
+
     # ###################### Conflicts ######################
 
     # I was unable to build root with any Intel compiler

